jenkins-on-ubuntu/azuredeploy-parameters.json

```diff
@@ -1,6 +1,6 @@
 {
   "storageAccountPrefix": {
-    "value": "jenkstore19"
+    "value": "jenkstore31"
```

Comment:
Yes that would be fine.  Should I make the change or are you doing that?

 

From: John Gossman [mailto:notifications@github.com] 
Sent: Thursday, April 30, 2015 8:49 AM
To: Azure/azure-quickstart-templates
Cc: dennisan
Subject: Re: [azure-quickstart-templates] changes to install script and clean up (#199)

 

In jenkins-on-ubuntu/azuredeploy-parameters.json <https://github.com/Azure/azure-quickstart-templates/pull/199#discussion_r29441771> :

> @@ -1,6 +1,6 @@
>  {
>    "storageAccountPrefix": {
> -    "value": "jenkstore19"
> +    "value": "jenkstore31"

Just jenkstore instead of jenkstore31?

—
Reply to this email directly or view it on GitHub <https://github.com/Azure/azure-quickstart-templates/pull/199/files#r29441771> .  <https://github.com/notifications/beacon/ABDe3re8vCbIEkJO8f_T0QD3opvqND4xks5oEkZogaJpZM4EMUgw.gif>